lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -308,7 +310,6 @@ void SessionStateEnforcer::m5g_remove_rules_and_update_upf(
   // remove all rules;
   MLOG(MINFO) << "Will be removing all associated rules of session id "
               << session->get_session_id();
-  session->remove_all_rules();

```

Comment:
why is this gone. Is `set_remove_all_pdrs` enough to remove all rules?

Reply:
Ya, its calling location moved to "m5g_complete_termination".  We need to remove the rule once get the confirmation (after expiry) we are deleting the PDR rules, as part of release this is changed.